CODE/style.css

```
@@ -0,0 +1,130 @@
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #d2d3d3;
+
+}
+
+/* pull QUote*/
+blockquote {
+    text-align: left;
+    margin: 0;
+}
+
+
+header {
+    background-image: url(images/maxresdefault.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+/*Title and Author*/
+h1 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 60px;
+    text-align: center;
+    color: #ce5454f8;
+}
+
+h2 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 45px;
+    text-align: center;
+    color: #ce5454f8;
+}
+
+h3 {
+    font-family: 'Oswald', sans-serif;
+    font-size: 30px;
+    text-align: left;
+}
+
+/*article text*/
+p1 {
+    columns: 100 px 1;
+    margin: -438px 20px 20px 270px;
+    float: left;
+    font-family: 'Libre Franklin', sans-serif;
+    font-size: 25px;
+    text-align: left;
+    line-height: 1.85;
+    padding: 30 20 50px;
```

Your padding rule is broken because it is missing units:
```suggestion
    padding: 30px 20px 50px;
```